packages/react/package.json

```diff
@@ -16,10 +16,14 @@
   },
   "scripts": {
     "build": "npm run clean && npm run compile && npm run copy:core-css && npm run copy:normalize-css",
+    "cypress": "cypress open",
+    "cypress:component": "cypress run --component --headless",
+    "cypress:ci": "npm run cypress:component",
     "clean": "rimraf dist",
     "compile": "npm run tsc",
     "tsc": "tsc -p .",
     "rollup": "rollup -c",
+    "lint:cypress": "eslint --config ./cypress/.eslintrc.json ./cypress/component",
```

Comment:
Can this npm script be added top level package.json file so it can be called in the CI pipeline work.

Reply:
Added :)